FLASK/attraction.py

```diff
@@ -0,0 +1,103 @@
+from database import DB
+from comment import Comment
+from rating import Rating
+
+class Attraction:
+    def __init__(self, id, name, location, image, description, rating, category):
+        self.id = id
```

Comment:
It would be better to make this  variable private and add getters/setters or decorate them  in appropriate way.